crates/re_arrow_store/src/store_gc.rs

```diff
@@ -613,25 +613,31 @@ impl IndexedBucketInner {
             }
 
             // col_row_id
-            let removed_row_id = col_row_id.swap_remove(row_index);
+            let Some(removed_row_id) = col_row_id.swap_remove(row_index) else {
+                continue;
+            };
             debug_assert_eq!(row_id, removed_row_id);
             dropped_num_bytes += removed_row_id.total_size_bytes();
 
             // col_time
-            let row_time = col_time.swap_remove(row_index);
+            let row_time = col_time.swap_remove(row_index).unwrap();
```

Comment:
No idea how that one unwrap survive, and even though it's unreachable, it gotta go